crates/astria-bridge-withdrawer/src/bridge_withdrawer/submitter/mod.rs

```diff
@@ -164,7 +164,7 @@ impl Submitter {
 
         // sign transaction
         let signed = unsigned.into_signed(signer.signing_key());
-        debug!(tx_hash = %telemetry::display::hex(&signed.sha256_of_proto_encoding()), "signed transaction");
+        debug!(tx_hash = &signed.id().get(), "signed transaction");
```

Comment:
IMO call this an ID like in all other places:
```suggestion
        debug!(transaction.id = %signed.id(), "signed transaction");
```